sdk/src/main/java/com/qonversion/android/sdk/internal/mappers/UserMapper.kt

```diff
@@ -0,0 +1,41 @@
+package com.qonversion.android.sdk.internal.mappers
+
+import com.qonversion.android.sdk.internal.extensions.mapping.getString
+import com.qonversion.android.sdk.internal.extensions.mapping.getDate
+import com.qonversion.android.sdk.internal.extensions.mapping.getMap
+import com.qonversion.android.sdk.public.Entitlement
+import com.qonversion.android.sdk.public.User
+import com.qonversion.android.sdk.public.UserPurchase
+
+class UserMapper internal constructor(
+    private val purchasesMapper: Mapper<UserPurchase>,
+    private val entitlementMapper: Mapper<Entitlement>
```

Comment:
We can use concrete mapper classes here. No need to use interface with declared generic type, I guess. Discussable

Reply:
I think it's a bad idea. We should use the interface everywhere we can. Otherway we can't replace implementation w/o replacing declaration right here. Maybe it's not such a popular case when you need to replace mapper implementation, but we should have a standardized approach.